docs/design.md

```diff
@@ -30,6 +30,15 @@ parameters:
     type: int
     default: 3306
 ```
+Or with an empty parameters field
+```yaml
+name: fusor/etherpad-apb
+description: Note taking web application
+bindable: true
+async: optional
+parameters: []
```

Comment:
Sure, a place holder works.  I agree with keeping an open issue until we have an example apb to replace this place holder.